posthog/demo.py

```diff
@@ -204,29 +206,21 @@ def _recalculate(team: Team) -> None:
         action.calculate_events()
 
 
-def demo(request):
+def demo(request: Request):
     user = request.user
     organization = user.organization
     try:
-        team = organization.teams.get(name=TEAM_NAME)
+        team = organization.teams.get(is_demo=True)
```

Comment:
Unhandled edge case of organization having multiple demo projects

Reply:
I don't think that case is currently possible.

I checked production and we seem to have 4 demo teams right now, all under different orgs.